src/app/shared-component/status-drawer.component.ts

```diff
@@ -10,7 +10,7 @@ import { WorkItem } from './../work-item/work-item';
 @Component({
 	selector   : 'status-drawer',
 	templateUrl: './status-drawer.component.html',
-	styleUrls  : ['src/app/shared-component/status-drawer.component.css'],
+	styleUrls  : ['/status-drawer.component.css'],
```

Comment:
Hey but it was not working for me. I was getting an exception. Then I went through [this](https://github.com/angular/angular-cli/issues/1501) post. The issue was with the white space before ':'. Instead of styleUrls : [] I did styleUrls: [] and it did work. I wonder, how is it working for you though.